library/account/src/main/kotlin/org/cru/godtools/account/GodToolsAccountManager.kt

```diff
@@ -117,3 +130,14 @@ class GodToolsAccountManager @VisibleForTesting internal constructor(
     internal suspend fun authenticateWithMobileContentApi() = activeProvider?.authenticateWithMobileContentApi(false)
         ?: Result.failure(AuthenticationException.NoActiveProvider)
 }
+
+// TODO Remove when GTO-Support-Util is updated
```

Comment:
you should be able to already use this from gto-support since the branch is using gto-support 4.2.3-SNAPSHOT